packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -58,11 +62,20 @@ const AdvertiserPage = () => {
                 setShouldShowPopup={advertiser_page_store.setShowAdPopup}
                 table_type={advertiser_page_store.counterparty_type === buy_sell.BUY ? buy_sell.BUY : buy_sell.SELL}
             />
-            <PageReturn
-                className='buy-sell__advertiser-page-return'
-                onClick={buy_sell_store.hideAdvertiserPage}
-                page_title={localize("Advertiser's page")}
-            />
+            <div className='advertiser-page__page-return-header'>
+                <PageReturn
+                    className='buy-sell__advertiser-page-return'
+                    onClick={buy_sell_store.hideAdvertiserPage}
+                    page_title={localize("Advertiser's page")}
+                />
+                <MobileWrapper>
+                    <AdvertiserPageDropdown
+                        is_dropdown_visible={is_dropdown_visible}
+                        onViewBlockModal={() => advertiser_page_store.setIsBlockUserModalOpen(true)}
+                        onViewDropdown={() => setIsDropdownVisible(prevState => !prevState)}
```

Comment:
Once you move this to the store it doesn't need to be passed as a prop, you can use it directly in AdvertiserPageDropdown component